src/main/scala/no/ndla/auth/AuthProperties.scala

```diff
@@ -33,7 +33,10 @@ object AuthProperties extends LazyLogging {
 
   val Environment = propOrElse("NDLA_ENVIRONMENT", "local")
 
-  lazy val Domain = Domains.get(Environment)
+  lazy val Domain = Map(
```

Comment:
Hvorfor legger vi til et eget map her i stedet for å oppdatere Domains i network?